data/food/ecosystemic_services/feed.json

```diff
@@ -1,151 +1,151 @@
-  },
-  "ground-beef-feedlot": {
-    "oilseed-feed": 6.1,
-    "barley-eu": 6.33,
-    "oats-glo": 12.68
-  },
-  "beef-without-bone": {
-    "silage-maize-fr": 2.79,
-    "soft-wheat-fr": 1.04,
-    "soybean-br-deforestation": 0.44,
-    "grazed-grass-temporary": 4.13,
-    "grazed-grass-permanent": 30.63
-  },
-  "beef-without-bone-organic": {
-    "silage-maize-organic": 0.63,
-    "soft-wheat-organic": 0.36,
-    "oats-organic": 0.26,
-    "triticale-organic": 1.3,
-    "alfalfa-organic": 1.11,
-    "grazed-grass-temporary": 8.45,
-    "grazed-grass-permanent": 28.34
-  },
-  "beef-with-bone": {
-    "silage-maize-fr": 2.23,
-    "soft-wheat-fr": 0.83,
-    "soybean-br-deforestation": 0.35,
-    "grazed-grass-temporary": 3.3,
-    "grazed-grass-permanent": 24.51
-  },
-  "milk": {
-    "silage-maize-fr": 0.349,
-    "soft-wheat-fr": 0.0857,
-    "soybean-br-deforestation": 0.0646,
-    "grazed-grass-temporary": 0.438,
-    "grazed-grass-permanent": 0.175
-  },
-  "milk-organic": {
-    "silage-maize-organic": 0.071,
-    "soft-wheat-organic": 0.0477,
-    "triticale-organic": 0.0384,
-    "grazed-grass-temporary": 0.579,
-    "grazed-grass-permanent": 0.814
-  },
-  "lamb-meat-without-bone": {
-    "soft-wheat-fr": 2.94,
-    "soybean-br-deforestation": 0.91,
-    "grazed-grass-temporary": 12,
-    "grazed-grass-permanent": 25
-  },
-  "lamb-meat-without-bone-organic": {
-    "soft-wheat-organic": 2.94,
-    "barley-organic": 1.5,
-    "grazed-grass-temporary": 14,
-    "grazed-grass-permanent": 67
-  },
-  "chicken-breast": {
-    "sweet-corn-fr": 1.9,
-    "soft-wheat-fr": 1.23,
-    "soybean-br-deforestation": 0.87,
-    "barley": 0.87
-  },
-  "chicken-breast-fr": {
-    "sweet-corn-fr": 1.9,
-    "soft-wheat-fr": 1.23,
-    "soybean-feed": 0.87,
-    "barley": 0.87
-  },
-  "chicken-breast-organic": {
-    "sweet-corn-organic-fr": 2.91,
-    "soft-wheat-organic": 0.63,
-    "soybean-organic": 0.87,
-    "barley": 0.87
-  },
-  "chicken-breast-br-max": {
-    "sweet-corn-br": 1.9,
-    "soft-wheat-fr": 1.23,
-    "soybean-feed": 0.87,
-    "barley": 0.87
-  },
-  "chicken-breast-fr-organic": {
-    "sweet-corn-organic-fr": 2.91,
-    "soft-wheat-organic": 0.63,
-    "soybean-organic": 0.87,
-    "barley": 0.87
-  },
-  "cooked-ham": {
-    "sweet-corn-fr": 0.74,
-    "soft-wheat-fr": 2.61,
-    "soybean-br-deforestation": 0.28,
-    "barley": 0.56
-  },
-  "cooked-ham-fr": {
-    "sweet-corn-fr": 1.11,
-    "soft-wheat-fr": 2.48,
-    "soybean-feed": 0.28,
-    "barley": 0.56
-  },
-  "cooked-ham-organic": {
-    "soft-wheat-organic": 0.72,
-    "soybean-organic": 0.58,
-    "barley-organic": 1.07,
-    "triticale-organic": 1.56,
-    "bean-organic": 1.46
+  "7ef7399a-1d56-4947-b773-9f6438a50fda": {
+    "40fa68b6-381e-4dd6-ba70-8b43cfdee64e": 0.485,
+    "a98b8776-a96d-48e9-b218-976f5452907a": 1.09,
+    "92d34545-6952-466e-bf6d-c1f1f6069b48": 0.204,
+    "bfdf1505-8426-499e-8286-a1e4b4920c7e": 0.274
+  },
```

Comment:
Sure but if we rely on aliases, we should add a check to verify that they are unique. For now, I don't think that this file is modified a lot, so not sure about the added value of a backend for it atm. But it would be a good entry point to start a backend.